src/Samhammer.Mongo/MongoDbConnector.cs

```diff
@@ -17,7 +17,7 @@ public class MongoDbConnector : IMongoDbConnector
 
         private readonly Lazy<Action> initConventions;
```

Comment:
i think we dont need lazy property for conventions when CreateMongoClient function is now safley only called once?